### PR TITLE
Fix issues with testnet dir, update docs

### PR DESCRIPTION
**Pull request by paulhauner**
## Issue Addressed

NA

## Proposed Changes

Fixes a bug introduced in #912 (I assume) that broke some of the `bn testnet ...` subcommands and also the `--testnet-dir` command when the hardcoded testnet is invalid (which is the case now).

Notably, this PR doesn't fix `bn testnet ...` it _only_ fixes `--testnet-dir` and then updates the docs to remove usages of `bn testnet ...` in favor of `--testnet-dir` + `lcli new-testnet`.

## Additional Info

A later PR can strip out all the old `bn testnet ...` subcommands.

### CODE REVIEW

File: book/src/local-testnets.md [hunk 1]
```diff
@@ -3,143 +3,104 @@
...
-```
+There is a TL;DR (too long; didn't read), followed by detailed steps if the
+TL;DR isn't adequate.
+
+##  TL;DR
 
 ```bash
-$ lighthouse vc testnet --help
+lcli new-testnet
+lcli interop-genesis
```
Comment (AgeManning) [inline book/src/local-testnets.md:22]:
```suggestion
lcli interop-genesis 16
```

**Comment by paulhauner** [comment]
Purging has been updated @michaelsproul :)

File: beacon_node/beacon_chain/src/builder.rs [hunk 2]
```diff
@@ -221,7 +221,7 @@ where
             .get::<PersistedBeaconChain>(&Hash256::from_slice(&BEACON_CHAIN_DB_KEY))
             .map_err(|e| format!("DB error when reading persisted beacon chain: {:?}", e))?
             .ok_or_else(|| {
-                "No persisted beacon chain found in store. Try purging the beacon chain database."
+                "No persisted beacon chain found in store. Try deleting the .lighthouse/beacon dir."
```
Comment (michaelsproul) [inline beacon_node/beacon_chain/src/builder.rs:224]: I think this should stay as it was

File: beacon_node/src/config.rs [hunk 3]
```diff
@@ -294,6 +291,39 @@ pub fn get_config<E: EthSpec>(
         client_config.network.discovery_address =
             Some("127.0.0.1".parse().expect("Valid IP address"))
     }
+
+    /*
+     * Load the eth2 testnet dir to obtain some addition config values.
```
Comment (michaelsproul) [inline beacon_node/src/config.rs:296]: addition -> additional

File: book/src/simple-testnet.md [hunk 4]
```diff
@@ -1,70 +1 @@
 # Simple Local Testnet
```
Comment (michaelsproul) [inline book/src/simple-testnet.md:1]: This file is just a heading, I think we should delete it?

Comment (paulhauner) [reply book/src/simple-testnet.md]: Thanks! `mdbook` has an annoying habit of creating a file if it's listed in `SUMMARY.md` but doesn't exist.

**Comment by paulhauner** [comment]
All comments addressed! :)

**Pull request merged**